app/services/piece-access-level-service.js

```diff
@@ -83,6 +83,24 @@ export default class PieceAccessLevelService extends Service {
     }
   }
 
+  /**
+   * Strengthen the access level of the piece to intern regering (unless the
+   * access level was already higher) and then update all the previous access
+   * levels.
+   */
+  async strengthenAccessLevel(piece) {
```

Comment:
Since the target access level is fixed in the method maybe better reflect it in the name? `strengthenAccessLevelToInternRegering`?